rhobs/alerting/data_plane/prometheus.latency_image_repository_provision_alerts.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-latency-image-repository-provision-alerting-rules
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: latency-image-repository-provision
+    interval: 1m
+    rules:
+    - alert: LatencyImageRepositoryProvision
+      expr: |
+        (
+          (
+            increase(redhat_appstudio_imagecontroller_image_repository_provision_time_bucket{le="+Inf"}[5m])
```

Comment:
I don't see this metric on AppSRE Grafana.

Reply:
other related PRs weren't merged yet
https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/92308
https://gitlab.cee.redhat.com/rhtap/docs/sop/-/merge_requests/69
https://github.com/redhat-appstudio/image-controller/pull/84